spacy/kb.pyx

```diff
@@ -286,11 +227,15 @@ cdef class KnowledgeBase:
             alias_entry.probs = probs
             self._aliases_table[alias_index] = alias_entry
 
+    def get_candidates(self, mention: Union[Span, str]) -> Iterator[Candidate]:
+        return self.get_alias_candidates(mention.text if isinstance(mention, Span) else mention)  # type: ignore
+
     def get_alias_candidates(self, str alias) -> Iterator[Candidate]:
         """
         Return candidate entities for an alias. Each candidate defines the entity, the original alias,
         and the prior probability of that alias resolving to that entity.
         If the alias is not known in the KB, and empty list is returned.
+        This particular function is optimized to work with the built-in KB functionality.
```

Comment:
Should we use the class name here so that this stays valid if we add other implementations?

Reply:
Actually, on reading it again it seems kinda redundant. This is part of the class that is the "built-in KB functionality". I think we could remove this line?

Reply:
Sure, that sounds fine.